python/GafferOSL/FocalBlur.gfr

```diff
@@ -0,0 +1,936 @@
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["cameraPath"], 'description', 'The scene path to the camera in `cameraScene` that you want to take settings from. If unset, defaults to the render camera set in the scene globals.' )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["cameraPath"], 'plugValueWidget:type', 'GafferSceneUI.ScenePathPlugValueWidget' )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["cameraPath"], 'scenePathPlugValueWidget:scene', 'cameraScene' )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["cameraPath"], 'scenePathPlugValueWidget:setNames', IECore.StringVectorData( [ '__cameras' ] ) )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["cameraPath"], 'scenePathPlugValueWidget:setsLabel', 'Show only cameras' )
+__children["FocalBlur"]["Expression"]["__in"]["p0"].setInput( __children["FocalBlur"]["cameraPath"] )
+__children["FocalBlur"]["Expression"]["__uiPosition"].setValue( imath.V2f( 775.942566, 29.858284 ) )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["depthChannel"], 'nodule:type', '' )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["depthChannel"], 'layout:section', 'Depth' )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["depthChannel"], 'layout:index', 18 )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["depthChannel"], 'description', 'Which image channel to use as the depth when computing the amount of blur.' )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["depthInterpretation"], 'nodule:type', '' )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["depthInterpretation"], 'layout:section', 'Depth' )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["depthInterpretation"], 'plugValueWidget:type', 'GafferUI.PresetsPlugValueWidget' )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["depthInterpretation"], 'preset:Unpremultiplied', 'unpremultiplied' )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["depthInterpretation"], 'preset:Premultiplied', 'premultiplied' )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["depthInterpretation"], 'preset:Filtered With Background Value ( Arnold )', 'filteredWithBackground' )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["depthInterpretation"], 'layout:index', 19 )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["depthInterpretation"], 'description', 'How the depth channel should be interpreted :\n\n- Premultiplied : Depth has been premultiplied by the alpha (typical when writing depth with a regular pixel filter).\n- Unpremultiplied : Depth has not been premultiplied by the alpha (typical when writing depth with a pixel filter such as "closest").\n- Premultiplied with background value (Arnold) : Depth includes a constant value for empty pixels, which needs to be subtracted before unpremultiplying. (typical with Arnold filtered depth which includes the camera far clip plane).' )
```

Comment:
We've ended up with "Premultiplied with background value (Arnold)" in this description, and "Filtered With Background Value ( Arnold )" in the preset name. We should standardise on one or the other.